lib/withdrawal.dart

```diff
@@ -221,6 +223,7 @@ class _WithdrawalFormScreenState extends State<WithdrawalFormScreen> {
             _recipientDescriptionController.text,
             tfCode);
         Navigator.pop(context);
+        Navigator.pop(context);
```

Comment:
likewise this should be moved after `MaterialPageRoute(
                    builder: (context) => CryptoWithdrawalDetailScreen(
                        withdrawal, widget.websocket)))` (line 227/30) which only executes **if** the withdrawal was created